### ISSUE TITLE
Font options don't seem to respect fontconfig alias families

### ISSUE THREAD

**Issue by somasis** [91861/wayst#24]
With this in the wayst configuration:

```text
font="monospace"
```

and in my fontconfig configuration files,

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
<fontconfig>
    <match target="pattern"> <!-- "serif" family -->
        <test qual="any"    name="family"><string>serif</string></test>
        <edit name="family" mode="prepend_first"><string>IBM Plex Serif</string></edit>
        <edit name="family" mode="prepend"><string>Noto Serif</string></edit>
        <edit name="family" mode="prepend"><string>Twemoji</string></edit>
    </match>

    <match target="pattern"> <!-- "sans-serif" family -->
        <test qual="any"    name="family"><string>sans-serif</string></test>
        <edit name="family" mode="prepend_first"><string>IBM Plex Sans</string></edit>
        <edit name="family" mode="prepend"><string>Noto Sans</string></edit>
        <edit name="family" mode="prepend"><string>Twemoji</string></edit>
    </match>
    <match target="pattern"> <!-- "monospace" family -->
        <test qual="any"    name="family"><string>monospace</string></test>
        <edit name="family" mode="prepend_first"><string>IBM Plex Mono</string></edit>
        <edit name="family" mode="prepend"><string>Noto Sans Mono</string></edit>
        <edit name="family" mode="prepend"><string>Twemoji</string></edit>
    </match>

    <alias binding="strong"> <!-- "emoji" family -->
        <family>emoji</family>
        <default><family>Twemoji</family></default>
    </alias>
</fontconfig>
```

`wayst` will report not being able to resolve the font.

```text
[error] Failed to load font 'monospace'
```

**Comment by 91861** [comment]
Fixed by 4303a20e015ec7f8b465397ed8e4f593927e1fe2.

**Comment by somasis** [comment]
Lovely. Thank you!
